src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -2548,6 +2647,326 @@ public final void addBatch() throws SQLServerException {
         loggerExternal.exiting(getClassNameLogging(), "executeLargeBatch", updateCounts);
         return updateCounts;
     }
+    
+    
+    private String parseUserSQLForTableNameDW(boolean hasInsertBeenFound, boolean hasIntoBeenFound) {
+        // As far as finding the table name goes, There are two cases:
+        // Insert into <tableName> and Insert <tableName>
+        // And there could be in-line comments (with /* and */) in between.
+        // This method assumes the localUserSQL string starts with "insert".
+        localUserSQL = localUserSQL.trim();
+        if (checkAndRemoveComments()) {
```

Comment:
Can't we just `checkAndRemoveComments` from the original query at the beginning?  Why do we have to do this inside every method?